### DISCUSSION TITLE
Cannot get twitter Oauth authentication to work

### DISCUSSION THREAD

**Discussion by joshtom** [nuxt-community/auth-module#1523] [Q&A]
I am trying  to authenticate a web app with twitter but I can't seem to find it In the docs and it's not  included in the demos provided. My question is, is there no direct support from nuxt auth again or I can just follow the same configuration as that of facebook and /  google. 

Thank you

**Comment by joshtom** [comment]
Hi @JoaoPedroAS51 Here is the issue I was referring to from issue [#1104](https://github.com/nuxt-community/auth-module/issues/1104)

Thank you!

**Comment by cchacholiades** [comment]
Hi @joshtom I am having the same problem. Have you managed to get this to work?

**Reply by joshtom** [reply]
Hi @cchacholiades 

I didn't get the answer I was looking for, So I later abandoned it.

**Comment by suleymangezsat** [comment]
Here is my solution for Twitter OAuth : https://codesandbox.io/s/eloquent-field-3ihhzh
Note: since requests to Twitter go through my backend, my backend has its own jwt mechanism (local scheme).
